.ddev/mautic-setup.sh

```diff
@@ -8,7 +8,7 @@ setup_mautic() {
     printf "Installing Mautic Composer dependencies...\n"
     composer install
 
-    cp ./.ddev/local.config.php.dist ./app/config/local.php
+    cp ./.ddev/local.config.php.dist ./local_config/local.php
```

Comment:
this naming does not seem right to me.
 I can imagine having the user configuration put aside, but why not name it just `/config` ? 
 * there is no such a directory
 * it is named the same way the `/app/config` is.
 * it does not contain the word **local** twice.
 * it feels super unnatural, I would never look for configuration in folder starting with `/local_`. so much for my autocomplete.

Reply:
this is addresses